README.md

```diff
@@ -4,41 +4,49 @@
 
 The above description, from the [Kubernetes homepage](https://kubernetes.io/), is centered on containerized _applications_. Yet, the Kubernetes metadata, objects, and visualizations (e.g., within Dashboard) are focused on container infrastructure rather than the applications themselves.
 
-The Application CRD [(Custom Resource Definition)](https://kubernetes.io/docs/concepts/api-extension/custom-resources/#customresourcedefinitions) and [Controller](https://kubernetes.io/docs/concepts/api-extension/custom-resources/#custom-controllers) in this project aim to change that in a way that's interoperable between many supporting tools.
+Applications will provides a way for you to aggregate individual Kubernetes objects (e.g. Services, Deployments, StatefulSets, Ingresses, and CRDs), and manage them as a group. Hopefully providing UIs that allows for the aggregation and display of all the objects in the Application. 
```

Comment:
> Applications will provides a way

This language is a bit confusing. Applications are already run inside Kuberentes and we are not trying to imply that this has a monopoly on the use of the term  Applications or that it introduces them. The title at the top of the README is a bit misleading now that I look at it this way.

This is to help with Application management and visualization in Kubernetes. Can we have another shot at the language?